runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesConfig.java

```diff
@@ -74,4 +74,16 @@ default boolean hideInventory()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "showAlchProfit",
+		name = "Show High Alchemy Profit/Loss",
+		description = "Show the profit/loss from casting high alchemy on items",
+		position = 5
+	)
+	default boolean showAlchProfit()
+	{
+		return true;
```

Comment:
This should be disabled by default because it is not useful in 90% of scenarios unless you are actually alching. If someone wants to enable it they can.